### PR TITLE
python2 -> python3 - CommonScripts -> anar

### PR DESCRIPTION
**Pull request by anara123**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
PrintErrorEntry move to python3

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [x] Yes
       - Further details:
   - [ ] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CommonScripts/Scripts/PrintErrorEntry/PrintErrorEntry_test.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import demistomock as demisto
+from CommonServerPython import * # noqa: F401
+
+
+def test_main(mocker):
+    from PrintErrorEntry import main
```
Comment (tkatzir) [inline Packs/CommonScripts/Scripts/PrintErrorEntry/PrintErrorEntry_test.py:6]:
```suggestion
    import PrintErrorEntry
```
And move it outside of the test_main method

File: Packs/CommonScripts/Scripts/PrintErrorEntry/PrintErrorEntry_test.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+    # test custom fields with short names
+    mocker.patch.object(demisto, 'args', return_value={
+        'message': 'this is error'
+    })
+    mocker.patch.object(demisto, 'results')
+    main()
```
Comment (tkatzir) [inline Packs/CommonScripts/Scripts/PrintErrorEntry/PrintErrorEntry_test.py:13]:
```suggestion
    PrintErrorEntry .main()
```

**Review by tkatzir — APPROVED** [review]
Some minor comments

**Review by tkatzir — APPROVED** [review]

**Comment by glicht** [comment]
Mypy error. See build

**Pull request merged**